lib/launcher.rb

```diff
@@ -1,9 +1,21 @@
+require 'bundler'
```

Comment:
Why is this needed? I don't see it used elsewhere

Reply:
Good point